Classes/ViewHelpers/Field/TextViewHelper.php

```diff
@@ -27,6 +27,9 @@ public function initializeArguments() {
 		$this->registerArgument('rows', 'int', 'Number of rows in editor', FALSE, 10);
 		$this->registerArgument('defaultExtras', 'string', 'FlexForm-syntax "defaultExtras" definition, example: "richtext[*]:rte_transform[mode=ts_css]"', FALSE, '');
 		$this->registerArgument('enableRichText', 'boolean', 'Shortcut for adding value of TS plugin.tx_flux.settings.flexform.rteDefaults to "defaultExtras"', FALSE, FALSE);
+		$this->registerArgument('renderType', 'string', 'Render type', FALSE, '');
```

Comment:
Last nit-pick: please describe the purpose of the arguments in their help text here - doesn't have to be the full monty, but should at least mention the most common possible values for render type and format arguments.